### PR TITLE
built input component

### PR DESCRIPTION
**Pull request by Rockyraven**

### CODE REVIEW

File: component/input/input.html [hunk 1]
```diff
@@ -5,74 +5,83 @@
...
-                style="width: 774px; height: 409px; border:0; transform: scale(1); overflow:hidden;"
                 sandbox="allow-scripts allow-same-origin">
             </iframe>
-            <div class="radio-checkbox-inputconatainer">
```
Comment (zabihshaik) [inline component/input/input.html:62]: Better to correct the typo for 'inputcontainer' instead of 'inputconatainer' as it may be used by others

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. 
Please check a single comment.

File: component/input/input.html (hunk 1 above)
Comment (Rockyraven) [reply component/input/input.html:62]: Hey i have already remove it
this red show removed code

Comment (zabihshaik) [reply component/input/input.html:62]: OK. Then it's Good to go.

**Review by zabihshaik — APPROVED** [review]

**Review by sagargse — APPROVED** [review]
all looks good to go

**Pull request merged**